### PR TITLE
EZP-28064: Add FieldLike Criterion support

### PR DESCRIPTION
**Pull request by wizhippo**

### CODE REVIEW

File: lib/Query/Common/CriterionVisitor/Field/FieldLike.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * This file is part of the eZ Platform Solr Search Engine package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\EzPlatformSolrSearchEngine\Query\Common\CriterionVisitor\Field;
```
Comment (alongosz) [inline lib/Query/Common/CriterionVisitor/Field/FieldLike.php]: Hmm, I missed one: there should be no blank line before `namespace` :( Sorry for the inconvenience.

**Comment by alongosz** [comment]
Thanks @wizhippo :)
Note: ezrobot review to be disregarded, checked manually.

**Comment by andrerom** [comment]
@wizhippo you can now rebase on top of master to double check we got all cs things right.

**Comment by andrerom** [comment]
@pspanja Seems the Fulltext tests fails on Solr 4, known? Should we just skip those tests in that case?

**Comment by andrerom** [comment]
Ok, issue is tests assumes scoring works on location search when they don't do that on Solr 4. So this is all good, merging, thanks @wizhippo!

**Pull request merged**